Tools/ros2/ardupilot_dds_tests/setup.py

```diff
@@ -21,11 +21,7 @@
     maintainer_email='maintainer@ardupilot.org',
     description='Tests for the ArduPilot AP_DDS library',
     license='GPL-3.0',
-    extras_require={
-        'test' : [
-            'pytest'
-        ]
-    },
+    extras_require={'test': ['pytest']},
```

Comment:
This is a crap change.  It makes the code less readable.

Please make black happy with the existing format which flake8 is happy with (is it just the space before the colon and the lack of comma after pytest?

Reply:
Looks like we can put a comma on line 26 and 27 to keep Black from collapsing this.